docs/subquery_network/node_operators/rewards.md

```diff
@@ -31,21 +27,15 @@ You can read more about how Delegators may pick Node Operators [here](../delegat
 
 Note, a Node Operator can [change the commission rate at any time](./setup/becoming-a-node-operator.md#6-configure-an-node-operator-commission-rate-nocr), but it takes an entire [Era](../introduction/era.md) for the new value to take effect.
 
-## Security & Performance Considerations
-
-Security and performance considerations are as follows.
-
-### Operator Wallets
-
-Secure storage of the wallet recovery seed phrase is highly recommended.
-
-### Firewalls
+## Security Considerations
 
-Node Operators need to keep security front of mind. Infrastructure security, in particular firewalls, should be implemented to prevent public exposure to personal ports.
+Security considerations for Node Operators are as follows.
 
-Secure passwords should be used by default and password rotation policies should be considered.
+- **Operator Wallets**: Secure storage of the wallet recovery seed phrase is highly recommended.
+- **Firewalls**: Node Operators need to keep security front of mind. Infrastructure security, in particular firewalls, should be implemented to prevent public exposure to personal ports.
```

Comment:
particularly avoid exposing Indexer Admin access to public. unsecured node runner may lose reward qualification